.devfilev2-intellij.yaml

```diff
@@ -0,0 +1,51 @@
+schemaVersion: 2.1.0
```

Comment:
this is in the wrong place, it should be in `scaffolder-templates/quarkus-template/skeleton` also it should be parameterized.

Reply:
Fixed